src/3p-frame.js

```diff
@@ -119,12 +121,16 @@ export function getIframe(parentWindow, parentElement, opt_type, opt_context) {
   const baseUrl = getBootstrapBaseUrl(parentWindow);
   const host = parseUrl(baseUrl).hostname;
   // Pass ad attributes to iframe via the fragment.
-  const src = baseUrl + '#' + JSON.stringify(attributes);
-  const name = host + '_' + attributes.type + '_' + count[attributes.type]++;
+  const name = JSON.stringify({
+    host,
+    type: attributes.type,
+    count: count[attributes.type],
```

Comment:
The counter is to ensure the iframe's window is not cached. https://github.com/ampproject/amphtml/pull/2955